src/status_im2/contexts/emoji_picker/view.cljs

```diff
@@ -0,0 +1,200 @@
+(ns status-im2.contexts.emoji-picker.view
+  (:require [oops.core :as oops]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo.theme]
+            [react-native.blur :as blur]
+            [react-native.core :as rn]
+            [react-native.gesture :as gesture]
+            [react-native.platform :as platform]
+            [reagent.core :as reagent]
+            [status-im2.contexts.emoji-picker.constants :as constants]
+            [status-im2.contexts.emoji-picker.data :as emoji-picker.data]
+            [status-im2.contexts.emoji-picker.style :as style]
+            [status-im2.contexts.emoji-picker.utils :as emoji-picker.utils]
+            [utils.debounce :as debounce]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.transforms :as transforms]))
+
+(defn- on-press-category
+  [id index active-category scroll-ref]
+  (reset! active-category id)
+  (some-> ^js @scroll-ref
+          (.scrollToIndex #js
+                           {:index    (emoji-picker.data/get-section-header-index-in-data index)
+                            :animated false})))
+
+(defn- handle-on-viewable-items-changed
+  [event active-category filtered-data]
+  (when-not @filtered-data
+    (let [viewable-item (-> (oops/oget event "viewableItems")
+                            transforms/js->clj
+                            first
+                            :item)
+          is-header?    (and (map? viewable-item) (:header? viewable-item))
+          section-key   (if is-header?
+                          (:id viewable-item)
+                          (emoji-picker.data/section-group (get-in (vec viewable-item) [0 :group])))]
+      (when (and (some? section-key) (not= @active-category section-key))
+        (reset! active-category section-key)))))
+
+(defn- get-item-layout
+  [_ index]
+  #js
+   {:length constants/get-item-height
+    :offset (* constants/get-item-height index)
+    :index  index})
+
+(defn- section-header
+  [{:keys [title]} _ _ {:keys [theme]}]
+  [quo/divider-label
+   {:tight?          false
+    :container-style (style/section-header theme)}
+   (i18n/label title)])
+
+(defn- emoji-item
+  [{:keys [unicode] :as emoji} col-index callback close]
+  (let [on-press          (fn []
+                            (when callback
+                              (callback unicode emoji))
+                            (close))
+        last-item-on-row? (= (inc col-index) constants/emojis-per-row)]
+    (fn []
+      [rn/pressable
+       {:style    (style/emoji-container last-item-on-row?)
+        :on-press on-press}
+       [rn/text
```

Comment:
should we not be using a quo component for this? 🤔 

Either way should we abstract this `text` component to be emoji-text or something? then all we need to pass is the value when using it

Reply:
By design, the emojis are Twemoji images. As Twemoji is not planned for MVP, we need to use the default system emoji for which we have to use the text component. 

The `text` component from `quo(2)` doesn’t have a `32px` (to match image size in design) font size in the config/typography. It's not worth having a temporary update in the `quo/text ' for this. We can use the Text from the RN and set the font size as per the size of the image (32px).  Post MVP, It will be easier for us to replace this with the image of Twemoji.